xml/System.Windows.Forms/HtmlWindowCollection.xml

```diff
@@ -46,7 +46,7 @@
   
  ]]></format>
     </remarks>
-    <related type="ExternalDocumentation" href="https://msdn.microsoft.com/library/default.asp?url=/workshop/browser/mshtml/reference/ifaces/framescollection2/framescollection2.asp?frame=true">IHTMLFramesCollection2 Interface</related>
+    <related type="ExternalDocumentation" href="https://docs.microsoft.com/en-us/">IHTMLFramesCollection2 Interface</related>
```

Comment:
These replacements are not correct. Not sure if my suggestion is correct but seems more appropriate.

```suggestion
    <related type="Article" href="https://docs.microsoft.com/previous-versions/windows/internet-explorer/ie-developer/platform-apis/aa752398(v=vs.85)">IHTMLFramesCollection2 Interface</related>
```